scripts/scratch_ingest/loom_ingest.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
```

Comment:
A module-level docstring here would help provide context for other engineers. 
 Example: https://github.com/broadinstitute/single_cell_portal/blob/master/scripts/manage_study.py.